### PR TITLE
356 1st round cleanupre org of pkgdown vignettes

### LINKED ISSUE #356: 1st Round cleanup/re-org of pkgdown vignettes
This issue involves the initial round of cleanup and reorganization of the pkgdown vignettes. The main objective is to remove duplicated content and restructure the table of contents on the pkgdown/article page.

For instance, the content of the following vignettes primarily focuses on the implementation of R functions, which can be incorporated as examples within the help files.
- [x] usage-expected-accrual.Rmd -> examples in `expected_accrual.R`
- [x] usage-expected-time.Rmd -> examples in `expected_time.R`
- [x] usage-summary-as-gt.Rmd -> examples in `summary.R` and `as_gt.R`
- [x] best-practice-npe-accuracy.Rmd -> examples in `gs_power_npe.R`

With this issue resolved, we will proceed with another round of cleanup for the vignettes, tentatively planned for around 2024 Q2. The focus of this cleanup will be on the content of the vignettes, including tasks such as adding new material, dividing certain vignettes into two, or merging multiple vignettes.

### PR DESCRIPTION
**Pull request by LittleBeannie**
Closes #356

### CODE REVIEW

**Comment by LittleBeannie** [comment]
Hi @nanxstats , do you have any idea of the CMD check error? It looks the problem lies in the `@examplesIf` of the `as_gt.R`. Could you please help with that?

File: R/as_gt.R [hunk 1]
```diff
@@ -280,17 +282,24 @@ as_gt.fixed_design <- function(x, title = NULL, footnote = NULL, ...) {
...
+#' # Example 6 ----
+#' # Usage of display_columns = ...
 #' # to select the columns to display in the summary table
 #' gs_power_wlr() %>%
 #'   summary() %>%
 #'   as_gt(display_columns = c("Analysis", "Bound", "Nominal p", "Z", "Probability"))
+#'
+#' # Example 7 ----
+#' # Customize the variables to be summarized for each analysis
+#' gs_power_wlr() %>%
```
Comment (nanxstats) [inline R/as_gt.R:301]: @LittleBeannie This is where the check error came from. This example is not complete.

File: R/expected_time.R [hunk 2]
```diff
@@ -76,6 +76,22 @@
 #'   target_event = xx$event, interval = c(.5, 1.5) * xx$time
 #' )
 #' }
+#'
+#' # Example 3 ----
+#' # In this example, we verify `expected_time()` by `ahr()`.
+#' \donttest{
+#' x <- ahr(
+#'   enroll_rate = enroll_rate, fail_rate = fail_rate,
+#'   ratio = ratio, total_duration = 20)
```
Comment (nanxstats) [inline R/expected_time.R:85]: Run styler on this file to get this example formatted.

File: R/gs_power_npe.R [hunk 3]
```diff
@@ -200,6 +200,31 @@
 #'   upar = (x %>% filter(bound == "upper"))$z,
 #'   lpar = -(x %>% filter(bound == "upper"))$z
 #' )
+#'
+#' # Different values of `r` and `tol` leads to differ numerical accuracy
```
Comment (nanxstats) [inline R/gs_power_npe.R:204]: leads -> lead?
differ -> different?

File: R/gs_power_npe.R [hunk 4]
```diff
@@ -200,6 +200,31 @@
...
+#' # Larger `r` and smaller `tol` give better accuracy, but leads to slow computation
```
Comment (nanxstats) [inline R/gs_power_npe.R:205]: leads -> lead?

File: R/gs_power_npe.R [hunk 5]
```diff
@@ -200,6 +200,31 @@
...
+#'   info1 = NULL,
+#'   info_scale = "h0_info",
+#'   upper = gs_spending_bound,
+#'   upar = list(sf = gsDesign::sfLDOF, total_spend = 0.025, param = NULL, timing = NULL),
+#'   lower = gs_b,
+#'   lpar = -rep(Inf, n_analysis),
+#'   test_upper = TRUE,
+#'   test_lower = FALSE,
+#'   binding = FALSE,
+#'   # Try different combination of (r, tol) with
```
Comment (nanxstats) [inline R/gs_power_npe.R:222]: combination -> combinations?

File: R/gs_power_npe.R [hunk 6]
```diff
@@ -200,6 +200,31 @@
...
+#'   # r in 6, 18, 24, 30, 35, 40, 50, 60, 70, 80, 90, 100
```
Comment (nanxstats) [inline R/gs_power_npe.R:223]: What does the comments "r in [some numbers]" and "tol in [two numbers]" actually trying to say?

File: _pkgdown.yml [hunk 7]
```diff
@@ -129,52 +129,45 @@ articles:
   contents:
   - gsDesign2
 
-- title: "NPH background"
+- title: "NPH technitical background"
```
Comment (nanxstats) [inline _pkgdown.yml:132]: technitical -> technical

File: _pkgdown.yml [hunk 8]
```diff
@@ -129,52 +129,45 @@ articles:
...
   desc: >
-    An introduction to non-proportional hazards.
+    Some technitical background introduction to non-proportional hazards, such as Z-score, B-values, and canonical form.
```
Comment (nanxstats) [inline _pkgdown.yml:134]: technitical -> technical

File: _pkgdown.yml [hunk 9]
```diff
@@ -129,52 +129,45 @@ articles:
...
   - articles/story-arbitrary-distribution
+  - articles/story-power-evaluation-with-spending-bound
+  - articles/story-compute-expected-events
+  - articles/story-info-formula
 
-- title: "Spending bound"
+- title: "Comparision of multiple NPH tests"
   desc: >
-    An introduction to spending bounds.
+    Comparison of power among different tests, such as regular logrank test, weighted logrank test with different weights (i.e., Fleming-Harrington, Magirr-Burman, etc.), and MaxCombo.
```
Comment (nanxstats) [inline _pkgdown.yml:147]: Use phrases like _that is_. Many people confuse _e.g._ and _i.e._

File: vignettes/articles/story-npe-background.Rmd [hunk 10]
```diff
@@ -23,14 +23,14 @@ knitr::opts_chunk$set(
 )
 ```
 
-```{r, message=FALSE, warning=FALSE}
+```{r, message=FALSE, warning=FALSE, echo=FALSE}
```
Comment (nanxstats) [inline vignettes/articles/story-npe-background.Rmd:26]: Why `echo = FALSE` for this vignette but not the others?

**Review by nanxstats — CHANGES_REQUESTED** [review]
Overall looking good, please see my specific comments below. The check error comes from the example code not being complete and should be easy to fix.

File: R/as_gt.R (hunk 1 above)
Comment (LittleBeannie) [reply R/as_gt.R:301]: Thanks for the check! It is exactly where the error is. Thank you!

File: R/gs_power_npe.R (hunk 3 above)
Comment (LittleBeannie) [reply R/gs_power_npe.R:204]: Corrected

File: R/gs_power_npe.R (hunk 4 above)
Comment (LittleBeannie) [reply R/gs_power_npe.R:205]: Correct

File: R/gs_power_npe.R (hunk 5 above)
Comment (LittleBeannie) [reply R/gs_power_npe.R:222]: Corrected

File: _pkgdown.yml (hunk 7 above)
Comment (LittleBeannie) [reply _pkgdown.yml:132]: Corrected

File: vignettes/articles/story-npe-background.Rmd (hunk 10 above)
Comment (LittleBeannie) [reply vignettes/articles/story-npe-background.Rmd:26]: This is try to library some utility R package for display purpose, so we can non-echo it

**Comment by LittleBeannie** [comment]
> Overall looking good, please see my specific comments below. The check error comes from the example code not being complete and should be easy to fix.

Thanks for the comments. They have been addressed and it is ready for review and merge.

**Review by nanxstats — APPROVED** [review]

**Pull request merged**